### DISCUSSION TITLE
ffmpeg: failed to download resource "netpbm"

### DISCUSSION THREAD

**Discussion by eterout** [Homebrew/discussions#3105] [Everyday usage]
Hello! I tried looking it up on my own but couldn't find any solution so this is my last resort. 
I'm trying to install ffmpeg (I have High Sierra) and I get this error:

`==> Checking out 4274
svn: E170013: Unable to connect to a repository at URL 'https://svn.code.sf.net/p/netpbm/code/stable'
svn: E230001: Server SSL certificate verification failed: certificate has expired
Error: ffmpeg: Failed to download resource "netpbm"
Failure while executing; `svn checkout https://svn.code.sf.net/p/netpbm/code/stable /Users/user/Library/Caches/Homebrew/netpbm--svn --quiet -r 4274` exited with 1. Here's the output:
svn: E170013: Unable to connect to a repository at URL 'https://svn.code.sf.net/p/netpbm/code/stable'
svn: E230001: Server SSL certificate verification failed: certificate has expired`

I'm not sure if this could be the right solution but i tried to use brew install netpbm command as well just to check and it was successful but didn't resolve the issue. What does this error mean and is there a way to resolve it? Thank you in advance!

**Comment by davidpiano** [comment]
I can't tell if it's a good answer or an approved answer, but one possible way of forcing the netpbm download to work is to do this:
`svn list https://svn.code.sf.net/p/netpbm/code/stable`

It will say this certificate has expired, and will give you choices like this:

`(R)eject, accept (t)emporarily or accept (p)ermanently? p`

If you choose p , to accept permanently, then you can go back to installing ffmpeg and hopefully it will work - or at least it won't fail because of netpbm.

**Reply by eterout** [reply]
thanks! after these commands it continued downloading. however at some point it stopped with this:

```
==> Downloading https://ftp.osuosl.org/pub/xiph/releases/flac/flac-1.3.4.tar.xz
Error: ffmpeg: Failed to download resource "flac"
Download failed: Homebrew-installed `curl` is not installed for: https://downloads.xiph.org/releases/flac/flac-1.3.4.tar.xz
```

**Reply by gromgit** [reply]
Looks like High Sierra's `curl` is too old. `brew install curl`, then try again.

**Reply by eterout** [reply]
ended up manually downloading ffmpeg and adding it to PATH and it all worked out, thank you for your help anyway!
